lib/cppcheck.cpp

```diff
@@ -131,16 +132,36 @@ namespace {
                     args += " " + v.get<std::string>();
             }
 
+            if(obj.count("python")) {
+                // Python was defined in the config file
+                if (obj["python"].is<picojson::array>()){
+                    return "Loading" + fileName +"failed. python must not be an array.";
```

Comment:
probably there should be space before and after filename

Reply:
Indeed there should be. Added as requested.